reference/constraints/Range.rst

```diff
@@ -394,15 +396,17 @@ minMessage
 **type**: ``string`` **default**: ``This value should be {{ limit }} or more.``
 
 The message that will be shown if the underlying value is less than the
-`min`_ option, and no `max`_ option has been defined (if both are defined, use
-`notInRangeMessage`_).
+`min`_ option, and **no** `max`_ option has been defined.
+
+If **both** are defined, use `notInRangeMessage`_.
 
 You can use the following parameters in this message:
 
 ===============  ==============================================================
 Parameter        Description
 ===============  ==============================================================
-``{{ limit }}``  The lower limit
+``{{ limit }}``  The lower limit (if only `min`_ is defined)
+``{{ min }}``    The lower limit (if both `min`_ and `max`_ are defined)
```

Comment:
This doesn't look right to me. I don't see where `{{ min }}` is actually used.

Reply:
it's used here https://github.com/symfony/validator/blob/d3a98390b77b2a8854c7b5d9058588d4b45794fb/Constraints/RangeValidator.php#L113